ui/raidboss/data/05-shb/raid/e3s.txt

```diff
@@ -20,7 +20,7 @@ hideall "--sync--"
 69.7 "Freak Wave" sync /:Leviathan:3FE6:/
 73.7 "Temporary Current" sync /:Leviathan:(3FEB|3FEA):/
 87.9 "Maelstrom" sync /:Leviathan:3FFA:/
-90.0  "--untargetable--"
+90.0 "--untargetable--"
```

Comment:
This is fine to change, but it might worth adding a test for it somewhere if you want it to be consistent across all timelines.

Reply:
I'm not sure if it makes sense to enforce this format.

I think the timelines are staying compatible with "ACT Timeline"? https://github.com/quisquous/cactbot/blob/master/ui/raidboss/timeline.js#L14 suggests this.

Reply:
Yes, that's the idea, although there's no reason cactbot timelines couldn't enforce more consistent formatting that is also backwards compatible, if that makes sense?